images/kali.yaml

```diff
@@ -1774,7 +1774,7 @@ actions:
     set -eux
 
     # Enable cloud-init units
-    systemctl enable cloud-init
+    systemctl enable cloud-init-main
```

Comment:
That's also the case on Ubuntu 24.10 BTW so we might eventually need to do `systemctl enable cloud-init || systemctl enable cloud-init-main` for every images.